WalletWasabi/WebClients/ChaumianCoinJoin/WasabiClient.cs

```diff
@@ -21,6 +22,29 @@ public WasabiClient(Uri baseUri, IPEndPoint torSocks5EndPoint = null) : base(bas
 		{
 		}
 
+		#region blockchain
+
+		public async Task<IEnumerable<string>> GetFiltersAsync(uint256 bestKnownBlockHash, int count)
```

Comment:
Also can you utilize this function in filter downloading?

Reply:
Yep.

Reply:
If we use this method in the IndexDownloader, how will we handle HttpStatusCode.NotFound? At the moment an exception will be thrown.